### ISSUE TITLE
Make Scout inclusive 

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2598]
Suggested actions:

- Adapt colors for users with visual impairment
- Check terminology and remove non-inclusive words (perhaps via a github action)
- Rename "master" to "main"
Other??

**Comment by mhkc** [comment]
I dont know if you do that at the moment but I think it would be a good idea to verify if the GUI behaves nice at different zoom levels or if we run into overflow and alignment issues.

Do you know if there is a need to support screen readers or similar aids for people with visual disabilities?

**Comment by dnil** [comment]
I haven't heard, but if this would be a site accessible to the public, we would definitely be required to support screen readers anyway. As it is now, I suppose it exists in a bit of limbo. I don't think e.g. our LIMS does. But as soon as a new coworker arrives - or develops vision problems - that could change. It is really good practice to have good text alts for any image, icon etc anyway.

As for sizes, we have discussed it a little, but not reached a conclusion. It would be very useful to have at least a min required size established, especially for like plugins with graphics.

**Comment by mhkc** [comment]
> As for sizes, we have discussed it a little, but not reached a conclusion. It would be very useful to have at least a min required size established, especially for like plugins with graphics.

I agree. Also I do believe this would have the highest impact on the current user base as peoples eyesight tends to worsen with age.
